src/docs/Contributing/Designers/NamingConvention/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Naming convention'
+order: 5
+---
+
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
```

Comment:
Mozaic is a product used by a lot of people on different teams, business units and even timezones.

 ~When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow.~
To have a minimum of organization, we need some ground rules that everyone that wants to contribute needs to follow.